compiler/damlc/tests/BUILD.bazel

```diff
@@ -483,6 +483,39 @@ da_haskell_test(
     ],
 )
 
+# Tests for multi-package
+da_haskell_test(
+    name = "multi-package",
+    srcs = ["src/DA/Test/DamlcMultiPackage.hs"],
+    data = [
+        "//daml-assistant:daml",
+        "//release:sdk-release-tarball",
+    ],
+    hackage_deps = [
+        "base",
+        "containers",
+        "directory",
+        "extra",
+        "filepath",
+        "process",
+        "regex-tdfa",
+        "tasty",
+        "tasty-hunit",
+        "text",
+        "time",
+    ],
+    main_function = "DA.Test.DamlcMultiPackage.main",
+    src_strip_prefix = "src",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//:sdk-version-hs-lib",
+        "//compiler/daml-lf-ast",
```

Comment:
I believe this isn't necessary

Reply:
that is, the test suite doesn't use any modules from `//compiler/daml-lf-ast` nor `//compiler/damlc/daml-opts:daml-opts-types` (I only noticed the latter now), so we can drop these from deps